LEGO1/pizza.cpp

```diff
@@ -13,7 +13,7 @@ Pizza::Pizza()
   this->m_unk88 = 0;
   this->m_unk8c = -1;
   this->m_unk98 = 0;
-  this->m_unk90 = 0x80000000;
+  this->m_unk90 = 0.0;
```

Comment:
I have a feeling this isn't a float. `0.0f` would ordinarily resolve to `0x0` in hex. The `80000000` could be valid for _negative_ `0.0f`, but that would be an extremely unusual thing for a developer to program intentionally. I think leaving this as a U32 is probably more accurate.